astrology.lic

```diff
@@ -39,6 +51,67 @@ class Astrology
     result.compact.select { |data| data['circle'] <= DRStats.circle }
   end
 
+  def check_moons
+    %w(xibar yavash katamba).each { |moon| bput("pow #{moon}", 'roundtime'); waitrt? }
```

Comment:
I'd prefer a linebreak instead of `;` (it's easy to miss the second action this way).